Umbraco.gitignore

```diff
@@ -1,12 +1,21 @@
 # Note: VisualStudio gitignore rules may also be relevant
 
+# =========================
+# Umbraco
+# =========================
 # Ignore unimportant folders generated by Umbraco
 **/App_Data/ClientDependency/
```

Comment:
Both the **/App_Data/ClientDependency/ and the **/App_Data/ExamineIndexes/ can be removed because that is not actually where those files are stored.